src/components/MarkerPopup/MarkerPopup.jsx

```
@@ -32,6 +34,22 @@ export const MarkerPopup = ({ place }) => {
     const handleMarkerClick = e => {
         setIsClicked(true);
     };
+
+    if (place.remark) {
```

ESLint

[error] 'place.remark' is missing in props validation

(react/prop-types)